package.json

```diff
@@ -1,15 +1,18 @@
 {
   "name": "tester-chrome-extension",
-  "version": "1.2.8",
-  "description": "The Rainforest Chrome Extension for Testers",
-  "main": "extension/js/main.js",
+  "version": "2.0.0",
+  "description": "The Rainforest Chrome Extension for Testers (internal JS)",
+  "main": "lib/index.js",
   "scripts": {
-    "test:mocha": "mocha test/tests.js --require mocha_babel",
+    "test:mocha": "mocha --require jsdom-global/register --compilers js:babel-register src/\\*\\*/__tests__/\\*.js",
     "test:lint": "eslint src",
     "test": "npm run test:mocha && npm run test:lint",
     "lint:fix": "eslint --fix src",
-    "watchTest": "mocha -w --require mocha_babel",
-    "build": "mkdir -p extension/js && browserify src/index.js -o extension/js/main.js -t [ babelify --presets [ es2015 ] ]"
+    "build:browser": "mkdir -p build && browserify src/bootstrap.js -d -o build/main.js -t [ babelify --presets [ es2015 ] ] -p [ minifyify --map main.js.map --output build/main.js.map ]",
+    "build:module": "mkdir -p lib && babel src --out-dir lib",
+    "build": "npm run build:browser && npm run build:module",
+    "postinstall": "rm -rf lib && npm run build:module",
```

Comment:
I don't think npm really makes sense for this because it's so specific. `prepublish` seems not to do anything when you're bringing it in direct from a repo? (Or at least it didn't when I tried it.)